src/callstack/lib_unwind.cr

```diff
@@ -1,3 +1,6 @@
+{% if flag?(:openbsd) %}
+  @[Link("stdc++")]
```

Comment:
Are we sure that libunwind doesn't exist as a seperate library to `libstdc++`?

Reply:
Yes